deviceStatus.go

```diff
@@ -20,8 +20,10 @@ func statusMetadata(d device.Interface) map[string]string {
 		wrpmeta.Field{From: "hw-serial-number", To: "/hw-serial-number"},
 		wrpmeta.Field{From: "hw-last-reboot-reason", To: "/hw-last-reboot-reason"},
 		wrpmeta.Field{From: "fw-name", To: "/fw-name"},
-		wrpmeta.Field{From: "last-reconnect-reason", To: "/last-reconnect-reason"},
-		wrpmeta.Field{From: "protocol", To: "/protocol"}).
+		wrpmeta.Field{From: "webpa-last-reconnect-reason", To: "/last-reconnect-reason"},
```

Comment:
@johnabass is there a way to accept either `webpa-last-reconnect-reason` or `last-reconnect-reason` and map them to `/last-reconnect-reason` ?

Reply:
Yes @schmidtw  we can add both fields, we just need to add this
```
wrpmeta.Field{From: "last-reconnect-reason", To: "/last-reconnect-reason"},
```

Logic is already written in wrp-go/blob/main/wrpmeta/metadata.go, if it is not able to find the filed then it will ignore the same,  problem here will be if we are getting both the fields in to convey header then the value will be overwritten  due to the same keys mentioned in `TO`

Reply:
I had added the support in latest commit